projects/dotcms/src/lib/utils/fieldsElementsTags.ts

```diff
@@ -0,0 +1,5 @@
+export enum FieldElementsTags {
```

Comment:
I don't think we're going to reuse this anywhere else but inside the class, so we can just move it there and not export it.